.github/workflows/pypi-cd.yml

```
@@ -162,18 +164,87 @@ jobs:
+                    
+                    echo "Downloading PyPy 3.10 for arm64"
+                    curl -LO "https://downloads.python.org/pypy/${PYPY_VERSION_3_10}.tar.bz2"
+                    sudo tar -xf "${PYPY_VERSION_3_10}.tar.bz2" --exclude="*/bin/python*" -C /opt
+                    
+                    echo "Downloading PyPy 3.11 for arm64"
+                    curl -LO "https://downloads.python.org/pypy/${PYPY_VERSION_3_11}.tar.bz2"
+                    sudo tar -xf "${PYPY_VERSION_3_11}.tar.bz2" --exclude="*/bin/python*" -C /opt
+                    
+                  elif [[ "$ARCH" == "x86_64" ]]; then
+                    PYPY_VERSION_3_11=pypy3.11-v7.3.19-macos_x86_64
+                    PYPY_VERSION_3_10=pypy3.10-v7.3.19-macos_x86_64
+                    PYPY_VERSION_3_9=pypy3.9-v7.3.16-macos_x86_64
+                    
+                    echo "Downloading PyPy 3.9 for x86_64"
+                    curl -LO "https://downloads.python.org/pypy/${PYPY_VERSION_3_9}.tar.bz2"
+                    sudo tar -xf "${PYPY_VERSION_3_9}.tar.bz2" --exclude="*/bin/python*" -C /opt
+                    
+                    echo "Downloading PyPy 3.10 for x86_64"
+                    curl -LO "https://downloads.python.org/pypy/${PYPY_VERSION_3_10}.tar.bz2"
+                    sudo tar -xf "${PYPY_VERSION_3_10}.tar.bz2" --exclude="*/bin/python*" -C /opt
+                    
+                    echo "Downloading PyPy 3.11 for x86_64"
+                    curl -LO "https://downloads.python.org/pypy/${PYPY_VERSION_3_11}.tar.bz2" 
+                    sudo tar -xf "${PYPY_VERSION_3_11}.tar.bz2" --exclude="*/bin/python*" -C /opt
+                  fi
+
+                  echo "/opt/${PYPY_VERSION_3_11}/bin:/opt/${PYPY_VERSION_3_10}/bin:/opt/${PYPY_VERSION_3_9}/bin" >> ${GITHUB_PATH}
+
+                  sudo chmod -R a+rx /opt/pypy*/bin
+                  sudo chmod -R a+rx /opt/pypy*/lib/*.dylib
+
+            ### ASYNC CLIENT ###
+
+            - name: Prepare the async client for deployment, vendor the glide-shared code
+              working-directory: ./python
+              run: cp -r glide-shared/glide_shared glide-async/python
+
+            - name: Build Python Async client wheels (linux)
               if: startsWith(matrix.build.NAMED_OS, 'linux')
               uses: PyO3/maturin-action@v1
               with:
                   working-directory: ./python/glide-async
                   target: ${{ matrix.build.TARGET }}
-                  args: --release --strip --out wheels -i ${{ github.event_name != 'pull_request' && 'python3.9 python3.10 python3.11 python3.12 python3.13 pypy3.9 pypy3.10 pypy3.11' || 'python3.12' }}
+                  args: --release --strip --out wheels -i ${{ (github.event_name != 'pull_request' && github.event_name != 'workflow_dispatch') && 'python3.9 python3.10 python3.11 python3.12 python3.13 pypy3.9 pypy3.10 pypy3.11' || 'python3.12' }}
```

```suggestion
                  args: --release --strip --out wheels -i ${{ (github.event_name != 'pull_request' && github.event_name != 'workflow_dispatch') && 'python3.9 python3.10 python3.11 python3.12 python3.13 pypy3.9 pypy3.10 pypy3.11' || 'python3.13' }}
```

why not 3.13, as it is the latest ver?